.circleci/config.yml

```diff
@@ -1013,7 +1019,7 @@ jobs:
     - run:
         name: "Gather Perf Data"
         command: |
-          "${CI_ROOT}/test-scripts/baseline/compare-and-update-baseline.sh"
+          "${CI_ROOT}/test-scripts/baseline/compare-and-update-baseline.sh" "true"
```

Comment:
This script is only called here with its only parameter set to `true`, could we remove this and cleanup both `.circleci/test-scripts/baseline/compare-and-update-baseline.sh` and `.circleci/test-scripts/baseline/main.py`, since it looks like this is now its default behavior and I don't see a point complicating the code if it is not going to be used.

Reply:
Done